### ISSUE TITLE
WP CLI commands that are missing

### ISSUE THREAD

**Issue by Mte90** [GlotPress/GlotPress-WP#958]
Right now this commands are missing:

- Upload a localized po to a specific project
- Create all the locales for a specific project

**Comment by ocean90** [comment]
See https://meta.trac.wordpress.org/browser/sites/trunk/wordpress.org/public_html/wp-content/plugins/wporg-gp-customizations/inc/cli/class-mass-create-sets.php for a `mass-create-sets` command.

**Comment by Mte90** [comment]
Probably we can migrate all this wp cli scripts inside glotpress https://meta.trac.wordpress.org/browser/sites/trunk/wordpress.org/public_html/wp-content/plugins/wporg-gp-customizations/inc/cli?order=name that right just for docs:

- generate a language pack 
- generate the pot file of wordpress core
- copy the translation set to a project to another one
- export core translation

**Comment by toolstack** [comment]
@mte90 don't forget that GP is not WP specific, so some of those functions would not become part of GP.

**Comment by Mte90** [comment]
Yes, was just to document in the ticket to have an idea about we have already as code available and what we need to do from scratch.

**Comment by Mte90** [comment]
For ClassicPress I done a forked version of the command that generate language pack by project.
It is quite ready (I have to change the path where generate them and probably also add a parameter for that) and I think that we can integrate it in Gp because is general purpose: https://github.com/Mte90/ClassicPress-Network/blob/gp-cli/wp-content/plugins/cp-gp-cli/inc/class-language-pack.php

Probably the other one that we can integrate for GP is the mass create sets avalaible there https://meta.trac.wordpress.org/browser/sites/trunk/wordpress.org/public_html/wp-content/plugins/wporg-gp-customizations/inc/cli/class-mass-create-sets.php that is also general purpose.

So in that way we will have already 2 new commands and we can start think to the others. In the case of the others avalaible on wp.org they are too much specific that infrastrcture so I don't think that they fit for this project.
